Dockerfile

```diff
@@ -23,34 +23,32 @@ RUN apt-get update && apt-get install --yes \
 # Add a "dockerdev" user with sudo capabilities
 # 1000 is the first user ID issued on Ubuntu; might
 # be different for Mac users. Might need to add more.
-RUN \
-     useradd -u ${UID} -ms /bin/bash -G sudo dockerdev \
-  && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers 
+RUN useradd -u ${UID} -ms /bin/bash -G sudo dockerdev && \
```

Comment:
There are no changes here other than reducing the number of `RUN` stanzas -- each `RUN` line introduces a new layer into the Docker container